### PR TITLE
adding support for a more flexible seed structure to support external urls

### PR DESCRIPTION
**Pull request by caridy**
this helps to control the yui version on the client.

### CODE REVIEW

**Comment by caridy** [comment]
An example of this is the use of the latest YUI version in the client side thru the seed configuration:

```
    [{
        "settings": [ "master" ],
        "yui": {
            "config": {
                "seed": [
                    "http://yui.yahooapis.com/3.8.1/build/yui-base/yui-base-min.js",
                    "http://yui.yahooapis.com/3.8.1/build/loader-base/loader-base-min.js",
                    "http://yui.yahooapis.com/3.8.1/build/loader-yui3/loader-yui3-min.js",
                    "loader-app",
                    "loader-app-base{langPath}"
                ],
                "gallery": "gallery-2013.01.16-21-05",
                "base": "http://yui.yahooapis.com/3.8.1/build/",
                "comboBase": "http://yui.yahooapis.com/combo?",
                "root": "3.8.1/build/"
            }
        }
    }]
```

In this case, 4 files will conform the seed. Alternative, those 3 YUI modules could be comboed, in which case we will have 2 files, one from YUI CDN and one from the app CDN to complete de Seed of the app. Happy hacking!

File: lib/app/addons/ac/deploy.server.js [hunk 1]
```diff
@@ -185,6 +169,61 @@ YUI.add('mojito-deploy-addon', function(Y, NAME) {
 
             // Add the boot script
             assetHandler.addAsset('blob', 'bottom', initializer);
+        },
+
+        /**
+         * Generate the YUI seed file to use on the client.
+        // This has to be done before any other scripts are
+        // added.
```
Comment (drewfish) [inline lib/app/addons/ac/deploy.server.js]: Small point:  would be nice if the comment leader was consistent.

**Comment by drewfish** [comment]
+1

File: lib/app/addons/ac/deploy.server.js (hunk 1 above)
Comment (caridy) [inline lib/app/addons/ac/deploy.server.js]: ops, fixing it.

**Pull request merged**